### PR TITLE
ETD-103 Auth & Code Cleanup

### PR DESCRIPTION
**Pull request by tolgatunaaa**
- Authentication Added with Stride
- Tests added related with Stride and page redirection
- Code cleaned from Hello world pages

### CODE REVIEW

**Review by Hari-M-K — APPROVED** [review]

File: conf/app.routes [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 POST       /                            controllers.EoriActionController.continueAction
 
-
 # Update EORI
 GET       /update/                      controllers.UpdateEoriController.showPage
 #POST      /update/                     controllers.UpdateEoriController.continueUpdateEori()
 
-GET        /hello                       controllers.HelloWorldController.helloWorld
-GET        /assets/*file                controllers.Assets.versioned(path = "/public", file: Asset)
+GET        /                            controllers.EoriActionController.showPage
```
Comment (ajayshirke22) [inline conf/app.routes:13]: Route for EoriActionController.showPage is two times added.

**Review by ajayshirke22 — COMMENTED** [review]
Only one observation, in app.routes double entry for EoriActionController.showPage.

File: conf/app.routes (hunk 1 above)
Comment (tolgatunaaa) [reply conf/app.routes:13]: Good Catch

**Review by ajayshirke22 — DISMISSED** [review]
Can proceed for merging.Thanks

**Review by Hari-M-K — APPROVED** [review]

**Pull request merged**